includes/TypesForSolids/GasSolids.js

```diff
@@ -0,0 +1,77 @@
+if (!RFEM) {
+	throw new Error("This script is only for RFEM, it works with solids.");
+}
+
+/*
+this.gas_solid.temperature = temperature (doesn't work??)
+*/
+
+/**
+* Creates gas solid
+* @class
+* @constructor
+* @param	{Number}	no			Index of gas solid
+* @param	{Array}		solid_list	List of solid indexes
+* @param	{String}	comment		Comment, can be undefined
+* @param	{Object}	params  	Gas solid's parameters, can be undefined
+* @returns	Created gas solid
+*/
+function GasSolid (no,
```

Comment:
I think better would be to add here parameters which you use in prototype GasSolidWithParameters and remove parameter solid_list. For assigment, Create prototype function AssingTo(solid_list)